app/models/coronavirus/timeline_entry.rb

```diff
@@ -16,4 +20,16 @@ def set_position
   def set_parent_positions
     page.make_timeline_entry_positions_sequential
   end
+
+private
+
+  def applies_to_uk_nations
+    uk_nations = %w[england northen_ireland scotland wales]
+
+    national_applicability.each do |nation|
+      unless uk_nations.include?(nation)
+        errors.add(:national_applicability, "is not a UK nation")
```

Comment:
Minor (as hopefully no-one ever sees this), this reads a bit strange when output. It'll say "National applicability is not a UK nation":

![Screenshot 2021-07-02 at 19 07 47](https://user-images.githubusercontent.com/282717/124313474-d0237b00-db68-11eb-97a1-13e380966136.png)

perhaps "had an invalid nation selected" ?